src/eradius_server.erl

```diff
@@ -391,10 +389,32 @@ inc_reply_counter(discnak, NasProp) ->
 inc_reply_counter(_Cmd, _NasProp) ->
     ok.
 
+inc_request_counter_accounting(accreq, NasProp, Ms, EncRequest) ->
+    #nas_prop{secret = Secret} = NasProp,
+    #radius_request{attrs = Attrs} = eradius_lib:decode_request(EncRequest, Secret),
```

Comment:
this would that we potentially decode the request twice. That would seriously impact performance.

Please find a way to implement this without the decoding. Maybe pass the decoded request in this or do the counter for specific accounting request later.

Reply:
Done